releng/com.b2international.snowowl.server.update/assembly/common/scripts/snowowl_delta_import.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 B2i Healthcare Pte Ltd, http://b2i.sg
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
`set -euo pipefail` is a common default incantation at the start of bash scripts: http://redsymbol.net/articles/unofficial-bash-strict-mode/ 

Short summary: 
* `set -e` exits the script if a command returns a non-zero exit code, doesn't move on to the next line in the script;
* `set -u` exits the script if a variable is referenced which is undefined; 
* `set -o pipefail` means the exit code from piped together processes only returns success if all processes in the pipe succeeded (default is only the last process status counts)

If it doesn't break any functionality, I'd recommend adding it here.